src/core/textrenderer/qgsfontmanager.cpp

```diff
@@ -203,65 +203,117 @@ QString QgsFontManager::urlForFontDownload( const QString &family, QString &matc
     QStringLiteral( "Advent Pro" ),
+    QStringLiteral( "Agdasima" ),
     QStringLiteral( "Aguafina Script" ),
+    QStringLiteral( "Akatab" ),
+    QStringLiteral( "Akaya Kanadaka" ),
+    QStringLiteral( "Akaya Telivigala" ),
     QStringLiteral( "Akronim" ),
+    QStringLiteral( "Akshar" ),
     QStringLiteral( "Aladin" ),
+    QStringLiteral( "Alata" ),
+    QStringLiteral( "Alatsi" ),
+    QStringLiteral( "Albert Sans" ),
     QStringLiteral( "Aldrich" ),
     QStringLiteral( "Alef" ),
     QStringLiteral( "Alegreya" ),
     QStringLiteral( "Alegreya SC" ),
     QStringLiteral( "Alegreya Sans" ),
     QStringLiteral( "Alegreya Sans SC" ),
+    QStringLiteral( "Aleo" ),
     QStringLiteral( "Alex Brush" ),
+    QStringLiteral( "Alexandria" ),
     QStringLiteral( "Alfa Slab One" ),
     QStringLiteral( "Alice" ),
     QStringLiteral( "Alike" ),
     QStringLiteral( "Alike Angular" ),
+    QStringLiteral( "Alkalami" ),
+    QStringLiteral( "Alkatra" ),
     QStringLiteral( "Allan" ),
     QStringLiteral( "Allerta" ),
     QStringLiteral( "Allerta Stencil" ),
+    QStringLiteral( "Allison" ),
     QStringLiteral( "Allura" ),
+    QStringLiteral( "Almarai" ),
     QStringLiteral( "Almendra" ),
     QStringLiteral( "Almendra Display" ),
     QStringLiteral( "Almendra SC" ),
+    QStringLiteral( "Alumni Sans" ),
+    QStringLiteral( "Alumni Sans Collegiate One" ),
+    QStringLiteral( "Alumni Sans Inline One" ),
+    QStringLiteral( "Alumni Sans Pinstripe" ),
     QStringLiteral( "Amarante" ),
     QStringLiteral( "Amaranth" ),
     QStringLiteral( "Amatic SC" ),
     QStringLiteral( "Amethysta" ),
+    QStringLiteral( "Amiko" ),
     QStringLiteral( "Amiri" ),
+    QStringLiteral( "Amiri Quran" ),
+    QStringLiteral( "Amita" ),
     QStringLiteral( "Anaheim" ),
     QStringLiteral( "Andada" ),
+    QStringLiteral( "Andada Pro" ),
     QStringLiteral( "Andika" ),
+    QStringLiteral( "Anek Bangla" ),
+    QStringLiteral( "Anek Devanagari" ),
+    QStringLiteral( "Anek Gujarati" ),
+    QStringLiteral( "Anek Gurmukhi" ),
+    QStringLiteral( "Anek Kannada" ),
+    QStringLiteral( "Anek Latin" ),
+    QStringLiteral( "Anek Malayalam" ),
+    QStringLiteral( "Anek Odia" ),
+    QStringLiteral( "Anek Tamil" ),
+    QStringLiteral( "Anek Telugu" ),
     QStringLiteral( "Angkor" ),
     QStringLiteral( "Annie Use Your Telescope" ),
     QStringLiteral( "Anonymous Pro" ),
     QStringLiteral( "Antic" ),
     QStringLiteral( "Antic Didone" ),
     QStringLiteral( "Antic Slab" ),
     QStringLiteral( "Anton" ),
+    QStringLiteral( "Antonio" ),
+    QStringLiteral( "Anuphan" ),
+    QStringLiteral( "Anybody" ),
+    QStringLiteral( "Aoboshi One" ),
     QStringLiteral( "Arapey" ),
     QStringLiteral( "Arbutus" ),
     QStringLiteral( "Arbutus Slab" ),
     QStringLiteral( "Architects Daughter" ),
     QStringLiteral( "Archivo" ),
     QStringLiteral( "Archivo Black" ),
     QStringLiteral( "Archivo Narrow" ),
+    QStringLiteral( "Are You Serious" ),
```

Comment:
Are _you_ serious? ;)

Reply:
Let's just hope no-one uses this in their maps :roll_eyes: 

![image](https://github.com/qgis/QGIS/assets/1829991/1b445dcb-39e1-4467-9c19-f1f248d94ad2)